scrapper.py

```diff
@@ -25,8 +28,8 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls: list, max_articles: int):
-        pass
+    def __init__(self, seed_urls: list, max_articles=1):
+        self.seed_urls = seed_urls
```

Comment:
other params should specified in the __init__ field . -> e.g. max_articles param